docs/Contributing.asciidoc

```diff
@@ -585,7 +585,13 @@ temporary PostgreSQL database used for testing. To do this step manually run
 `t/test_postgresql /dev/shm/tpg` to initialize a temporary PostgreSQL database
 and export the environment variable as instructed by that script.
 It is also possible to run a particular test, for example
-`prove t/api/01-workers.t`.
+`prove t/api/01-workers.t`. When using `prove` directly, make sure an English
+locale is set (e.g. `export LANG=C` before initializing the database and running
+`prove`).
+
+To keep the test database running after executing tests with the `Makefile`, add
+`KEEP_DB=1` to the make arguments. To access the test database, use
```

Comment:
we already mention `KEEP_DB` elsewhere but I don't mind

Reply:
I know. But in this context it is relevant as well. (We likely should really merge the two documentation sections at some point.)